package.json

```diff
@@ -68,11 +68,14 @@
     "loader.js": "^4.0.1",
     "lodash": "^4.17.4",
     "lodash.omit": "^4.5.0",
-    "moment": "^2.18.1",
     "pagination-pager": "2.4.2",
     "regenerator": "^0.8.46"
   },
   "devDependencies": {
-    "ember-simple-auth": "^1.4.0"
+    "ember-cli-bootstrap-datetimepicker": "^0.5.0",
+    "ember-simple-auth": "^1.4.0",
+    "eonasdan-bootstrap-datetimepicker-npm": "4.17.42",
```

Comment:
Is this deprecated?

Reply:
my guess is that ember-cli-bootstrap-datetimepicker might be a wrapper around the eonasdan one?